### PR TITLE
Improve readability of version info message

### PR DESCRIPTION
**Pull request by TerryKern**
Instead of: 

> You are already using composer version 1.10.15 (1 channel).

it shows: 

> You are already using composer version 1.10.15 (1.x channel).

### CODE REVIEW

File: src/Composer/Command/SelfUpdateCommand.php [hunk 1]
```diff
@@ -177,7 +177,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         if (Composer::VERSION === $updateVersion) {
-            $io->writeError(sprintf('<info>You are already using composer version %s (%s channel).</info>', $updateVersion, $versionsUtil->getChannel()));
+            $io->writeError(sprintf('<info>You are already using composer version %s (%s.x channel).</info>', $updateVersion, $versionsUtil->getChannel()));
```
Comment (stof) [inline src/Composer/Command/SelfUpdateCommand.php:180]: wouldn't this cause issues when the channel is `preview` for instance ?

Comment (TerryKern) [reply src/Composer/Command/SelfUpdateCommand.php:180]: What are the possible contents of ``$versionsUtil->getChannel())``?

Comment (TerryKern) [reply src/Composer/Command/SelfUpdateCommand.php:180]: Is there an alternative to ``getChannel()`` that produces a more readable output?

Comment (stof) [reply src/Composer/Command/SelfUpdateCommand.php:180]: The list of channels is at https://github.com/composer/composer/blob/99a5469b5e3d956d3b7e014147186bbd3678276e/src/Composer/SelfUpdate/Versions.php#L24

Comment (TerryKern) [reply src/Composer/Command/SelfUpdateCommand.php]: @stof What do you think of the new solution?

**Comment by TerryKern** [comment]
@Seldaek I retargetted it to 1.10

**Comment by Seldaek** [comment]
Thanks, sorry looking at this again, I'd say it probably makes sense to tweak also line 211 the output for when an update actually happens.

**Comment by TerryKern** [comment]
> Thanks, sorry looking at this again, I'd say it probably makes sense to tweak also line 211 the output for when an update actually happens.

@Seldaek Done :)

**Comment by Seldaek** [comment]
Thanks

**Comment by TerryKern** [comment]
@Seldaek Would you consider marking this as a valid Hacktoberfest PR?
https://hacktoberfest.digitalocean.com/hacktoberfest-update

**Comment by Seldaek** [comment]
I opted in the repo so this should be valid now.

**Comment by TerryKern** [comment]
@Seldaek Thank you :)

**Pull request merged**